app/views/my_facilities/_recent_updates.html.erb

```diff
@@ -29,7 +37,7 @@
       Feb <b class="d-block">2022</b>
     </div>
     <p>
-      The <b>Simple Android app</b> now allows users to edit and add Health IDs (such as BP Passports and NHID) from the patient's profile screen. In India, this makes it easier to add NHIDs to follow-up patients.
+      The <b>Simple Android app</b> now allows users to edit and add Health IDs (such as BP Passports<% if CountryConfig.current[:name] == "India" %> and NHID<% end %>) from the patient's profile screen.<% if CountryConfig.current[:name] == "India" %> In India, this makes it easier to add NHIDs to follow-up patients.<% end %>
```

Comment:
Should we pluralize `and NHIDs`?

In the last sentence, I reckon we can remove the clause "In India..." because we only show it in India now.